rs/nns/governance/src/neuron_store.rs

```diff
@@ -1320,11 +1320,26 @@ impl NeuronStore {
 #[allow(unused)] // This line will be removed soon...
 pub fn prune_some_following(
     neuron_store: &mut NeuronStore,
-    mut next: Bound<NeuronId>,
-    mut carry_on: impl FnMut() -> bool,
+    next: Bound<NeuronId>,
+    carry_on: impl FnMut() -> bool,
 ) -> Bound<NeuronId> {
     let now_seconds = neuron_store.now();
+    groom_some_neurons(
+        neuron_store,
+        |neuron| {
+            neuron.prune_following(now_seconds);
+        },
+        next,
+        carry_on,
+    )
```

Comment:
I spun out groom_some_neurons. Then, I used groom_some_neurons to (very concisely) implement backfill_some_voting_power_refreshed_timestamps.